src/View/Widget/RadioWidget.php

```diff
@@ -32,20 +41,49 @@ public function render(array $data, ContextInterface $context)
             'inline' => false,
         ];
 
+        $templates = $this->_templates->get();
+        $customize = [];
+
         if ($data['inline']) {
-            $this->_templates->add(['radioContainer' => '{{content}}']);
+            $this->_inline = $data['inline'];
+            if ($templates['formGroup'] === '{{label}}{{input}}') {
+                $this->_templates->add(['radioFormGroup' => '{{label}}<div class="radio-inline-wrapper">{{input}}</div>']);
+            }
         }
 
-
-        $templates = $this->_templates->get();
-        $customize = [];
-        if ($templates['nestingLabel'] === '{{hidden}}<label{{attrs}}>{{input}}{{text}}</label>') {
+        if (!$data['inline'] && $templates['nestingLabel'] === '{{hidden}}<label{{attrs}}>{{input}}{{text}}</label>') {
```

Comment:
Same comment as above. This check will fail if the template is modified by user.